src/containers/PodcastSeries/components/PodcastSeriesForm.tsx

```diff
@@ -131,6 +132,18 @@ const PodcastSeriesForm = ({
     setSavedToServer(true);
   };
 
+  const validateMetaImage = ([width, height]: [
+    number,
+    number,
+  ]): FormikErrors<PodcastFormValues> => {
+    if (width !== height) {
+      return { coverPhotoId: t('validation.podcastImageShape') };
+    } else if (width < 1400 || width > 3000) {
```

Comment:
Disse er bestemt av apple/spotify/podcast-folk sant?
Stemmer for å dra de ut i en konstant med et litt forklarende navn evt en kommentar om navnet blir langt og stygt.
Ser for meg at jeg kan bruke litt tid på å skjønne hvor disse kommer fra i fremtiden (evt ikke skjønne at de kommer fra et sted og bare endre de).

Reply:
Eg stussa også på dette, men fant forklaringa i apple sin spesifikasjon. Kan være lurt å legge inn disse i const med navn itunes-standard-minimum-witdth eller tilsvarende.